pages/linux/wipefs.md

```diff
@@ -7,10 +7,14 @@
 
 `sudo wipefs {{/dev/sdX}}`
 
-- Wipe all available signatures for specified device:
+- Wipe all available signatures for specified device level (no recursion to sub-level block layers):
```

Comment:
Which level? block level?

Reply:
"block level" is a bit fuzzy now that I think on it again. My understanding is Linux block layers stack up, such that `/dev/sda` would be the full device, then `/dev/sda1` would be the first partition, etc. But technically, perhaps, we should rather say "device and all partitions", given there's other block stuff, like LVM, which doesn't behave this way.

Also, if one does `wipefs /dev/sda*` and there's a partition with an LVM in the partition, I'm guessing wipefs will only clear up the LVM metadata, but not the filesystem metadata with the logical volumes, etc. So there's only so much wipefs does. To truly recurse to all block layers would require a script.

I'll submit another commit to the pull to word this better. My "block layer" wording is too generic.

(Apologies for a tardy reply.)